content/en/docs/Features/multi-cluster.md

```diff
@@ -1,24 +1,56 @@
+
+For multi-cluster configurations, Kiali provides a unified view and management of your mesh across clusters.
+
+## Graph View: Cluster and Namespace Boxing
 
-Kiali v1.29 introduces experimental support for advanced deployment models. The recommended Kiali deployment model is to deploy an instance of Kiali along with each Istio primary control plane.  Each Kiali instance will work as it has in the past, with the configured Kubernetes, Prometheus and Jaeger instances.  It will concern itself with the istio config managed by the local primary.  But, there are two new additions:
+Istio provides cluster names in the traffic telemetry for multi-cluster installations. The Kiali graph can use this information to better visualize clusters and namespaces. The Display menu offers two multi-cluster related options: Cluster Boxes and Namespace Boxes. When enabled, either separately or together, the graph will generate boxes to help identify the relevant nodes and edges, and to see traffic traveling between them.
+
+Each box type supports selection and provides a side-panel summary of traffic. Below we see a Bookinfo traffic graph for when Bookinfo services are deployed across the East and West clusters. The West cluster box is selected. You see traffic for all services and workloads across both clusters. Single cluster configurations will show some traffic across clusters but not all.
+
+![Multi-cluster traffic graph](/images/documentation/features/multi-cluster-traffic-graph.png "Multi-cluster traffic graph")
 
 ## List View: Mesh Discovery
-Kiali will attempt to discover the clusters configured in the mesh.  And it will identify the *home* cluster, meaning the cluster on which it is installed and from which it presents its traffic, traces and Istio config.  In the following example there are two clusters defined in the mesh, _Kukulcan_ and _Tzotz_.  Kukulcan is identified as the home cluster in three places: the browser tab (not shown), the masthead, and with a star icon in the clusters list:
 
-![Mesh list view](/images/documentation/features/multi-cluster-mesh-v1.29.png "Mesh list view")
+Kiali will show cluster information for all clusters in the mesh. In single-cluster configurations, Kiali will attempt to discover the clusters configured in the mesh. It will identify the home cluster, meaning the cluster on which it is installed and from which it presents its traffic, traces and Istio config. In the following example there are two clusters defined in the mesh, Kukulcan and Tzotz. Kukulcan is identified as the home cluster in three places: the browser tab (not shown), the masthead, and with a star icon in the clusters list:
```

Comment:
Maybe we can update the image here? 
So the example cluster names are the same. Also here, it is included the Kiali column.

![image](https://github.com/kiali/kiali.io/assets/49480155/4bf36a26-0b6d-4c58-a083-841b6764498f)